### ISSUE TITLE
WCAG 2.2 Violation - There are controls present that convey state-related information both programatically and within their accessible names.

### ISSUE THREAD

**Issue by ivrouwe-ubc** [ubc/wp-comments-chat-ui#13]
### Issue

There are controls within the interface that present state -related information in their accessible names (instead of solely programmatically), resulting in non-descriptive accessible names.

#### This violates the following WCAG (Web Content Accessibility Guidelines) 2.2 Success Criteria:

- [2.4.6 Headings and Labels](https://www.w3.org/TR/WCAG22/#headings-and-labels)

### Instances

- #29 

### User Impact

Some users may be confused by the names of the controls and/or unclear on the current state of the controls.

#### Further information:

1. ["G131: Providing descriptive labels | WAI | W3C"](https://www.w3.org/WAI/WCAG22/Techniques/general/G131)

#### Using the Chrome developer tools:

1. Inspect the control using Chrome's developer tools.
2. Activate the "Accessibility" tab (for the element) within the developer tools.
3. Under "Computed Properties", notice that the "Name" value includes state information.
4. Looking at the control itself (in the HTML view of the Inspector), notice that there are ARIA attributes set on the control to convey its state programmatically (meaning that the state information being included in the control's accessible name is redundant, and may be confusing to users).

#### Using a screen reader (such as VoiceOver, NVDA, JAWS, or TalkBack):

1. Activate the screen reader.
2. Navigate to the control.
3. Notice that the control's accessible name includes state -related information (alongside the state information that is already being conveyed programatically via the inclusion of various ARIA attributes on the control.

### How To Fix The Issue

Do not include state information as part of a control's accessible name. Instead, allow the ARIA attributes that you've already included to convey (and communicate) the state of the control to assistive technology users.

#### WCAG 2.2 Conformant Code Example
Not available/not applicable.
